### ISSUE TITLE
Routes.js files in develop don't have coverage

### ISSUE THREAD

**Issue by uo289321** [Arquisoft/wiq_es05b#167]
Even with the tests done, sonarCloud keeps saying that they have 0% coverage. Tried to change the name, still 0%.

**Comment by augustocristian** [comment]
Hi everyone!
I checked a few minutes ago and the pipeline reported good coverage.
![image](https://github.com/Arquisoft/wiq_es05b/assets/49859508/373b1e4a-f17b-48f0-9a03-8c27fa9fed87)
Please provide me more details about the problem, but looks like something related to:
 https://github.com/kulshekhar/ts-jest/issues/542#issuecomment-700103076

**Comment by augustocristian** [comment]
After the group have implemented the changes suggested by @pglez82, I've been examining the local Sonar logs. Removing the relative paths resolved the warnings, but the code coverage remains lower than expected. It seems that the files are being properly indexed for analysis, as evidenced by these two examples:
![Example 1](https://github.com/Arquisoft/wiq_es05b/assets/49859508/601b7b98-14b9-47da-a14f-5cf87de665a1)
![Example 2](https://github.com/Arquisoft/wiq_es05b/assets/49859508/0d5fbedd-b9d9-48bb-838b-22d6b870747a)

**Comment by algarfer** [comment]
Yes, I activated the debug mode on e2f394a37fa7c39878dc1793a1bcb5760029c4f7.

Also, I have tested generating jest configuration files and changing the paths of sonar, but these don't seem to fix it at all.

I've also noticed that there are some inconsistencies in the coverage reports:

![image](https://github.com/Arquisoft/wiq_es05b/assets/58784148/d4e873a9-5fb2-4852-a0bc-45c0d9ae7557)

Further information can be found [here](https://github.com/orgs/Arquisoft/discussions/67)
